forui/lib/src/theme/font/font_data.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+/// A class that holds the font data.
+class FFontData {
+  /// The size that corresponds to the `fontSize` property on [TextStyle].
+  final double fontSize;
+
+  /// The letter spacing that corresponds to the `letterSpacing` property on [TextStyle].
+  final double letterSpacing;
+
+  /// The word spacing that corresponds to the `wordSpacing` property on [TextStyle].
+  final double wordSpacing;
+
+  /// The height that corresponds to the `height` property on [TextStyle].
+  final double height;
```

Comment:
I feel like these names are really misleading. Shouldn't they be scaling factors?

Also  we should first explain that these properties are used for multiplicative scaling before then mentioning that it is similar to a property in TextStyle in a 2nd sentence.

Reply:
That's a good catch. I decided to suffix the variables with `Scalar` as it is similar to definition in physics and scalar vector multiplication in linear algebra.